replicator/active_replicator_config.go

```diff
@@ -15,13 +17,19 @@ const (
 // ActiveReplicatorConfig controls the behaviour of the active replicator.
 // TODO: This might be replaced with ReplicatorConfig in the future.
 type ActiveReplicatorConfig struct {
-	ID string
-	// Type of replication: PushAndPull, Push, or Pull
+	ID     string
+	Filter string
+	// Since represents the sequence we're going to perform the replication from.
+	Since uint64
```

Comment:
I expect this needs to be a SequenceID, to cover cases where the remote (or checkpoint) returns a compound sequence, either due to backfill or skipped sequences.